c2cgeoform/views/abstract_views.py

```diff
@@ -230,22 +230,24 @@ def _sort_query(self, query, sort, order):
             if field.id() == sort:
                 criterion = field.sort_column()
                 if order == 'desc':
-                    criterion = desc(criterion)
+                    query = query.order_by(criterion.desc())
+                else:
+                    query = query.order_by(criterion)
                 criteria.append(criterion)
```

Comment:
criteria is no more used
seems that criterion can be replaced by field.sort_column()